examples/webgpu_reflection_blurred.html

```diff
@@ -0,0 +1,234 @@
+
+			import { OrbitControls } from 'three/addons/controls/OrbitControls.js';
+
+			import { GUI } from 'three/addons/libs/lil-gui.module.min.js';
+
+			import Stats from 'three/addons/libs/stats.module.js';
+
+			let camera, scene, renderer;
+			let model, mixer, clock;
+			let controls;
+			let stats;
+			let gui;
+
+			init();
+
+			async function init() {
+
+				camera = new THREE.PerspectiveCamera( 50, window.innerWidth / window.innerHeight, 0.25, 30 );
+				camera.position.set( - 2.5, 2, 2.5 );
+				camera.lookAt( 0, .4, 0 );
+
+				scene = new THREE.Scene();
+				scene.backgroundNode = hue( normalWorld.y.mix( 0, color( 0x0066ff ) ).mul( .1 ), time );
+
+				const waterAmbientLight = new THREE.HemisphereLight( 0xffffff, 0x0066ff, 10 );
+				scene.add( waterAmbientLight );
+
+				clock = new THREE.Clock();
+
+				// animated model
+
+				const gltfLoader = new GLTFLoader();
+				gltfLoader.load( 'models/gltf/Michelle.glb', function ( gltf ) {
+
+					model = gltf.scene;
+					model.children[ 0 ].children[ 0 ].castShadow = true;
+
+					mixer = new THREE.AnimationMixer( model );
+
+					const action = mixer.clipAction( gltf.animations[ 0 ] );
+					action.play();
+
+					scene.add( model );
+
+				} );
+
+				// textures
+
+				const textureLoader = new THREE.TextureLoader();
+
+				const uvMap = textureLoader.load( 'textures/uv_grid_directx.jpg' );
+				uvMap.colorSpace = THREE.SRGBColorSpace;
+
+				// uv map for debugging
+
+				const uvMaterial = new THREE.MeshStandardNodeMaterial( {
+					map: uvMap,
+					side: THREE.DoubleSide
+				} );
+
+				const uvMesh = new THREE.Mesh( new THREE.PlaneGeometry( 2, 2 ), uvMaterial );
+				uvMesh.position.set( 0, 1, - 3 );
+				scene.add( uvMesh );
+
+				// circle effect
+
+				const drawCircle = Fn( ( [ pos, radius, width, power, color, timer = time.mul( .5 ) ] ) => {
+
+					// https://www.shadertoy.com/view/3tdSRn
+
+					const dist1 = length( pos );
+					dist1.assign( fract( dist1.mul( 5.0 ).sub( fract( timer ) ) ) );
+					const dist2 = dist1.sub( radius );
+					const intensity = pow( radius.div( abs( dist2 ) ), width );
+					const col = color.rgb.mul( intensity ).mul( power ).mul( max( sub( 0.8, abs( dist2 ) ), 0.0 ) );
+
+					return col;
+
+				} );
+
+				const circleFadeY = positionWorld.y.mul( .7 ).oneMinus().max( 0 );
+				const animatedColor = mix( color( 0x74ccf4 ), color( 0x7f00c5 ), positionWorld.xz.distance( 0 ).div( 10 ).clamp() );
+				const animatedCircle = hue( drawCircle( positionWorld.xz.mul( .1 ), 0.5, 0.8, .01, animatedColor ).mul( circleFadeY ), time );
+
+				const floorLight = new THREE.PointLight( 0xffffff );
+				floorLight.colorNode = animatedCircle.mul( 50 );
+				scene.add( floorLight );
+
+				// reflection
+
+				const roughness = uniform( .9 );
+				const radius = uniform( 0.2 );
+
+				const reflection = reflector( { resolution: 1, depth: true, bounces: false } ); // 0.5 is half of the rendering view
```

Comment:
The example looks awesome! 

The only thing I would suggest to change is the default resolution of the reflector. On my macMini M2 Pro, the current example runs with 30 FPS. Using the half resolution for the reflection, the frame rate is back to 60 FPS. Since the reflections are blurred, you barely see a difference in quality anyway.

Reply:
If we set resolution to `0.5` on a non-retina screen the reflection becomes pixelated:

![Screenshot 2025-05-16 at 5 04 41 PM](https://github.com/user-attachments/assets/89692ab7-aeec-44e6-b536-e6f53f018bde)

Instead of `0.5` maybe something like `1 / devicePixelRatio` would work better?